### ISSUE TITLE
RCS1074 triggered when constructor has attributes

### ISSUE THREAD

**Issue by gtbuchanan** [JosefPihrt/Roslynator#962]
**Product and Version Used**: 
Roslynator 4.1.1

**Steps to Reproduce**:
1. Create a class or record with an empty constructor
2. Add an attribute (e.g. ReSharper's `UsedImplicitlyAttribute`) to the constructor

**Actual Behavior**:
```csharp
public sealed class MyClass
{
    [UsedImplicitly]
    public MyClass() // Triggers RCS1074
    {
    }
}
```

**Expected Behavior**:
[RCS1074](https://github.com/JosefPihrt/Roslynator/blob/main/docs/analyzers/RCS1074.md) should not be triggered when an attribute is added to the empty constructor because this makes the constructor differ from the default constructor.

**Comment by JosefPihrt** [comment]
Hi,

can you please tell me what is the full name the attribute (the name + containing namespace)?

**Comment by gtbuchanan** [comment]
@JosefPihrt It is `JetBrains.Annotations.UsedImplicitlyAttribute`. You can find it [here](https://github.com/JetBrains/JetBrains.Annotations/blob/main/src/Annotations.cs#L481).
